microsoft/iot-cardboard-js
# OAT: Limit reference count to 200 to mitigate perf issues

### Summary of changes 🔍 
In order to not create an extremely bad experience for users, we are limiting the number of references that can exist in an ontology to 200. This is the point where things start to get pretty laggy.

![image](https://user-images.githubusercontent.com/57726991/218891078-b7f64080-820c-45a1-b347-9fe3bd4e8dc8.png)


### Testing 🧪
Make an ontology with a bunch of models and make lots o' relationships between them. Then try to add another one and see it throw the error.
- Create new model should work
- Create relationship wiht existing model - blocked
- Create new model + relationship (by dragging to empty space) - blocked
- Import folder/file that pushes the count over.
